### PR TITLE
feat(inscription): inscription for dr and dd

### PR DESCRIPTION
**Pull request by gonzalezad1**

### CODE REVIEW

File: packages/api/controllers/user/signup.js [hunk 1]
```diff
@@ -107,6 +118,14 @@ const postSignup = async (req, res) => {
           .where("id", user.id);
         await createUserTis(req.body, user);
         break;
+      case "direction":
+      case "direction_national":
+      case "direction_regional":
+      case "direction_departemen": {
```
Comment (tglatt) [inline packages/api/controllers/user/signup.js]: direction_departemental ?

File: packages/api/controllers/user/signup.js [hunk 2]
```diff
@@ -107,6 +118,14 @@ const postSignup = async (req, res) => {
           .where("id", user.id);
         await createUserTis(req.body, user);
         break;
+      case "direction":
```
Comment (kwiss) [inline packages/api/controllers/user/signup.js]: Only direction and after in the function direction_*** switch ?

File: packages/api/controllers/user/signup.js [hunk 3]
```diff
@@ -60,6 +61,23 @@ const createUserTis = (body, user) =>
         })
     )
   );
+const createDirection = (body, user) =>
+  Direction.query().insert({
+    user_id: user.id,
+    department_id:
+      body.type === "direction_departemental" ? body.department_id : null,
+    region_id: body.type === "direction_regional" ? body.region_id : null
```
Comment (tglatt) [inline packages/api/controllers/user/signup.js]: strange identation

File: packages/api/controllers/user/signup.js [hunk 4]
```diff
@@ -107,6 +118,13 @@ const postSignup = async (req, res) => {
           .where("id", user.id);
         await createUserTis(req.body, user);
         break;
+      case "direction_national":
+      case "direction_regional":
+      case "direction_departemental": {
+        await createRole(user, "direction");
```
Comment (tglatt) [inline packages/api/controllers/user/signup.js]: we might have to create two roles for all cases:
```javascript
case "direction_national":
 await createRole(user, "direction");
 await createRole(user, "direction_national");
 await createDirection(req.body, user);
case "direction_regional":
 await createRole(user, "direction");
 await createRole(user, "direction_regional");
 await createDirection(req.body, user);
case "direction_departemental":
  await createRole(user, "direction");
  await createRole(user, "direction_departemental");
  await createDirection(req.body, user);
```

**Pull request merged**